ldap/ldap.go

```diff
@@ -68,6 +76,11 @@ type Config struct {
 	UserDNSearchBaseDistNames []BaseDNInfo
 	UserDNSearchFilter        string
 
+	// Additional attributes to fetch from the user DN search.
+	UserDNAttributes string
+	// this is a computed value from UserDNAttributes
+	UserDNAttributesList []string
```

Comment:
maybe s/UserDNAttributesList/userDNAttributesList ?

Reply:
Need this to be exported

Reply:
are these immutable values @donatello ?

Reply:
Yes immutable

Reply:
ok this is fixed now.